config/initializers/upload_types.rb

```diff
@@ -0,0 +1,19 @@
+UPLOAD_TYPES = [
+    *CSV_TYPES_TABLES
+].freeze
+
+def klass_name(upload)
```

Comment:
i think this method would be available in the whole app.  I think the old implementation wasn't too complex to need a method.  Also, there aren't any instances of the klass being a string, is there?  This seems unnecessary

Reply:
The array being added in https://github.com/department-of-veterans-affairs/gibct-data-service/pull/751 will have String values as there won't be a class to associate with it.

Reply:
removed the methods